dockerfiles/run/guice/cassandra-ldap/destination/conf/healthcheck.properties

```diff
@@ -0,0 +1,30 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+#  This template file can be used as example for James Server configuration
+#  DO NOT USE IT AS SUCH AND ADAPT IT TO YOUR NEEDS
+
+# Configuration file for Periodical Health Checks
+
+# Read https://james.apache.org/server/config-healthcheck.html for further details
+
+# Optional. Initial delay in seconds before PeriodicalHealthChecks starts. Default value is 60
+# healthcheck.initial.delay=60
```

Comment:
IMO this configuration parameter is useless. can't we use either the period or just 0 ?

Reply:
May be changing it to an option to disable the periodical logging would be better?